config/rbac/auth_proxy_service.yaml

```diff
@@ -4,11 +4,12 @@ metadata:
   labels:
     control-plane: controller-manager
   name: controller-manager-metrics-service
-  namespace: system
+  namespace: openstack
```

Comment:
We've kept this as `system` in the other operators.  Was there a specific reason it became `openstack` here?

Reply:
@abays No no reason to keep openstack here, it's just miss, /me will update.